libs/application-generic/src/utils/sanitize-control-values.ts

```diff
@@ -21,6 +21,7 @@ function sanitizeEmptyInput<T_Type>(input: T_Type, defaultValue: T_Type = undefi
 }
 
 export function sanitizeRedirect(redirect: InAppRedirectType | undefined) {
+  // TODO: There is a bug here, if the redirect doesn't contain both a url and a target it is removed from the new controlValues
```

Comment:
We must fix this so as not to have issues if the endpoints are used by the API and not the Dashboard that always sends a full object for redirectURls and actions.